app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :user
```

Comment:
- [ ] Its will be very important to stick with the author_id as foreign key just as you have done in the Post model and as required in the pproject requrement so that your app can work as expected. So i think you should update your belong to relationship to:

`belongs_to :author, class_name: 'User'`